src/UnitTests/TestBase.cs

```diff
@@ -63,8 +63,8 @@ public class NodeInfo {
 
         public Window LaunchApp(string exeFileName, string args, string rootElementName) {
             ProcessStartInfo info = new ProcessStartInfo();
-            info.FileName = exeFileName;
-            info.Arguments = args;
+            info.FileName = "\"" + exeFileName   + "\"";
+            info.Arguments = "\"" + args + "\"";
```

Comment:
technically Arguments can contain more than one, in which case this would not be the right fix.  I realize LaunchApp is never used right now with more than one argument, but I'd prefer this set of double quotes to be moved to the LaunchNotepad call where the "filename" is passed as an argument.